code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -536,9 +536,6 @@
 	color = "#C8A5DC"
 	metabolization_rate = 0.3
 	overdose_threshold = 35
-	addiction_chance = 1
-	addiction_chance = 10
-	addiction_threshold = 10
```

Comment:
Alternatively, you could add `minor_addiction = TRUE` which will not make it have any effects, but still give the funny messages.